analyzer/compliance-analyzer/src/main/java/eu/fasten/analyzer/complianceanalyzer/ComplianceAnalyzerPlugin.java

```diff
@@ -0,0 +1,278 @@
+                // project's info
+                String input1 = " -e s#java-plugin#java-plugin-" + artifactID + "#";
+                String input2 = " -e s#url#" + repoUrl + "#";
+                String command = "sed" + input1 + input2 + " docker/job.yaml";
+
+                Process p = Runtime.getRuntime().exec(command);
+
+                BufferedReader stdInput = new BufferedReader(new InputStreamReader(p.getInputStream()));
+                BufferedReader stdError = new BufferedReader(new InputStreamReader(p.getErrorStream()));
+                String s = null;
+
+                File f = new File(PATCHED_JOB_FILE_NAME);
+                FileWriter fr = new FileWriter(f);
+                BufferedWriter br  = new BufferedWriter(fr);
+
+                // write the sed ouput
+                while ((s = stdInput.readLine()) != null) {
+                    br.write(s);
+                    br.newLine();
+                }
+
+                br.close();
+
+                // read any errors from the attempted command
+                System.out.println("Here is the standard error of the command (if any):\n");
+                while ((s = stdError.readLine()) != null) {
+                    System.out.println(s);
+                }
+            }  catch (IOException ex) {
+                logger.info(ex.toString());
+                ex.printStackTrace();
+            }
+        }
+
+        public void ApplyK8sJob() {
+            try {
+                // Deploy ConfigMap
+                V1ConfigMap configMap = Yaml.loadAs(new File("docker/master-config.yaml"), V1ConfigMap.class);
+                V1ConfigMap deployedConfigMap = new CoreV1Api().createNamespacedConfigMap(K8S_NAMESPACE, configMap, null, null, null);
+                System.out.println("Deployed ConfigMap: " + deployedConfigMap);
+
+                // Deploy Service
+                V1Service service = Yaml.loadAs(new File("docker/service.yaml"), V1Service.class);
+                V1Service deployedService = new CoreV1Api().createNamespacedService(K8S_NAMESPACE, service, null, null, null);
+                System.out.println("Deployed Service: " + deployedService);
+
+                // Define and apply a qmstr kubernetes job
+                Yaml.addModelMap("v1", "Job", V1Job.class);
+
+                File file = new File(PATCHED_JOB_FILE_NAME);
+                V1Job yamlJob = (V1Job) Yaml.load(file);
+                logger.info("Job definition: " + yamlJob);
+                BatchV1Api api = new BatchV1Api();
+
+                V1Job result = api.createNamespacedJob(K8S_NAMESPACE, yamlJob, null, null, null);
+                System.out.println(result);
+
+                // delete job
+                /*
+                V1Status deleteResult = api.deleteNamespacedJob(yamlJob.getMetadata().getName(),
+                KUBERNETES_NAMESPACE, null, null, null, null, null, new V1DeleteOptions());
+                logger.info(deleteResult);
+                s*/
```

Comment:
Why is it commented out? Delete if not needed

Reply:
Deleted in https://github.com/fasten-project/fasten/commit/7975246450da52793713310dc6123c14c876daf5.